src/plugins/intel_gna/tests/unit/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ addIeTargetTest(
             PRIVATE
                 ngraphFunctions
                 gmock
-                commonTestUtils_s
+                common_test_utils_s
```

Comment:
Please check why we need to link `common_test_utils_s` to here? Do we really need both `common_test_utils_s` and `common_test_utils`?

Reply:
Yes common_test_utils_s  is required for unitTestUtils